cafe_branch2_menu.md

```diff
@@ -1,4 +1,8 @@
 <html>
 <title>Joe’s café - branch2 menu</title>
+<ul>
+<li>signature_latte</li>
```

Comment:
Has special character at cafe, please correct it.

Reply:
updated